orkaudio/Makefile.am

```diff
@@ -6,7 +6,7 @@ AUTOMAKE_OPTIONS = foreign 1.4
 sbin_PROGRAMS = orkaudio
 orkaudio_LDADD =
 orkaudio_LDFLAGS = -lxerces-c -llog4cxx -lorkbase -lsndfile -l@speex_lib@ -lapr-1 -lssl -lcrypto -lpthread -ldl 
-orkaudio_SOURCES = 	OrkAudio.cpp
+orkaudio_SOURCES = 	OrkAudio.cpp filters/LiveStream/LiveStreamServerProxy.cpp filters/LiveStream/LiveStreamServer.cpp
```

Comment:
Is this actually required? Given that they already being compiled as submodule.

Reply:
these changes are non needed anymore